metricbeat/helper/interfacer.go

```diff
@@ -32,6 +33,7 @@ type ModuleConfig struct {
 	MetricSets []string `config:"metricsets"`
 	Enabled    bool     `config:"enabled"`
 	Selectors  []string `config:"selectors"`
+	Filter     []filter.FilterConfig
```

Comment:
What about having in libbeat support for modules as we have for beat? This generic libbeat module should contain all the common configuration options for all modules. For now we have modules only in Metricbeat, but we are thinking anyway to have modules for other Beats as well.